docs/source/toc.yaml

```diff
@@ -52,5 +52,24 @@ subtrees:
                     title: Deploy the HugeCTR Model with Triton
                   - file: examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb
                     title: Deploy the TensorFlow Model with Triton
+      - file: api_overview
+        title: API Documentation
+        entries:
+          - file: api_core/merlin.dag
+            title: merlin.dag
+          - file: api_dataloader
+            title: merlin.loader
+          - file: api_core/merlin.io
+            title: merlin.io
+          - file: api_models
+            title: merlin.models
+          - file: api_core/merlin.schema
+            title: merlin.schema
+          - file: api_systems
+            title: merlin.systems
+          - file: api_nvtabular
+            title: nvtabular
+          - file: ./api_transformers4rec/modules
```

Comment:
This certainly looks correct.  The HTML gets built: https://nvidia-merlin.github.io/Merlin/review/pr-1041/api_transformers4rec/modules.html

The dot-slash look innocuous, but maybe that's the issue.